packages/insomnia/src/ui/components/present-users.tsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
-import { useParams } from 'react-router-dom';
+import { useParams, useRouteLoaderData } from 'react-router-dom';
 
 import { getAccountId } from '../../account/session';
 import { usePresenceContext } from '../context/app/presence-context';
+import { ProjectsLoaderData } from '../routes/project';
 import { AvatarGroup } from './avatar';
 
 export const PresentUsers = () => {
   const { presence } = usePresenceContext();
-  const { projectId, workspaceId } = useParams() as {
-    workspaceId: string;
-    projectId: string;
-  };
+  const { workspaceId } = useParams() as { workspaceId: string };
+  const projectData = useRouteLoaderData('/project') as ProjectsLoaderData | null;
 
   if (!presence) {
```

Comment:
```suggestion
  if (!presence || !projectData?.activeProject) {
```
get rid of the unwanted optional chains below